package.json

```diff
@@ -8,15 +8,17 @@
 	},
 	"scripts": {
 		"test": "yarn",
-		"build": "./bin/build.sh || npx play-sound beep.mp3",
+		"bootstrap": "npx lerna bootstrap",
```

Comment:
no need to use `npx` when a module is called from an NPM script. Also I would suggest adding `lerna bootstrap --force-local` as it ensures local packages are correctly linked

Reply:
Done, thanks @danielelisi